examples/controller_test.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+Controller module for uArm Controller.
```

Comment:
Let's create another repo for the gateway to communicate with this uArm Controller.